package.json

```diff
@@ -21,13 +21,14 @@
     "autosize": "^1.18.13",
     "bootstrap": "3.1.1",
     "events": "*",
+    "extract-text-webpack-plugin": "^1.0.1",
     "haste-resolver-webpack-plugin": "^0.2.2",
     "i18next": "3.4.4",
     "i18next-xhr-backend": "1.1.0",
+    "jQuery-Impromptu": "trentrichardson/jQuery-Impromptu#v6.0.0",
     "jquery": "~2.1.1",
     "jquery-contextmenu": "*",
     "jquery-i18next": "1.1.0",
-    "jQuery-Impromptu": "trentrichardson/jQuery-Impromptu#v6.0.0",
```

Comment:
Yes, sure. This probably was caused because I use Yarn instead of npm (yarn is definitely faster) and it could change the order of dependencies in `package.json`